integration/okhttp/src/main/java/com/bumptech/glide/integration/okhttp/OkHttpStreamFetcher.java

```diff
@@ -89,11 +89,13 @@ public void cancel() {
     // TODO: call cancel on the client when this method is called on a background thread. See #257
   }
 
+  @android.support.annotation.NonNull
```

Comment:
Yeah it's ok to add the dependency, the main library requires the annotations anyway so it's not a new dependency for applications.